jinseok/service/userService.js

```diff
@@ -15,6 +18,14 @@ const signUp = async (
   pwValidationCheck(password);
   emailValidationCheck(email);
 
+  const saltRounds = parseInt(process.env.SALT_ROUNDS);
+  const makeHash = async (password, saltRounds) => {
+    return await bcrypt.hash(password, saltRounds);
+  };
+
+  const hashedPassword = await makeHash(password, saltRounds);
+  password = hashedPassword;
```

Comment:
```suggestion
  const hashedPassword = await bcrypt.hash(password, saltRounds);
```
makeHash 함수는 내부적으로 어떠한 로직도 갖고 있지 않으므로 곧바로 hashedPassword에서 작성될 수 있도록 해주세요.

또한 plain password에 hashedPassword를 덮어씌우는게 아닌, createUser method에 hashedPassword를 인자로 넘겨주면 됩니다.